scripts/base/protocols/dns/main.zeek

```diff
@@ -429,6 +443,14 @@ event dns_request(c: connection, msg: dns_msg, query: string, qtype: count, qcla
 	c$dns$qtype_name  = query_types[qtype];
 	c$dns$Z           = msg$Z;
 
```

Comment:
Does AD get parsed in a request? I only did a quick skim of the RF, but it sounds like it should only be valid in a response.

Reply:
In the canonical RFC for AD, it was left undefined; RFC 6840 clarified that the AD bit can be set by the request or response
https://www.rfc-editor.org/rfc/rfc6840#section-5.7